include/bls12-381/g.hpp

```diff
@@ -60,7 +60,7 @@ class g1
 
     auto operator<=>(const g1&) const = default;
    
-    static std::optional<g1> multiExp(const std::vector<g1>& points, const std::vector<std::array<uint64_t, 4>>& scalars, std::function<void()> yield = std::function<void()>());
+   static std::optional<g1> multiExp(std::span<const g1> points, std::span<const std::array<uint64_t, 4>> scalars, std::function<void()> yield = std::function<void()>());
```

Comment:
Should we remove the unused `yield` ?

Reply:
Well, it is referenced in the implementation, but I agree it doesn't seem very useful. I don't really know enough about why it was added in the first place to have an opinion on this, can we leave this decision for later?

Reply:
Yes, certainly.